static/js/main.js

```diff
@@ -464,9 +468,16 @@ function initializeHotkeys() {
     });
 }
 
+function removeKeydownSideEffects() {
+    document.querySelectorAll('input[type="text"]').forEach((input) => {
+        input.setAttribute('onkeydown', 'event.stopPropagation()');
+    });
+}
+
 function main() {
     fetchVersion();
     initializeHotkeys();
+    removeKeydownSideEffects();
```

Comment:
Move this call inside initializeHotkeys().

It's nicer when functions deal with their own fallout.